x/leveragelp/keeper/msg_server_close_positions.go

```diff
@@ -34,6 +35,11 @@ func (k msgServer) ClosePositions(goCtx context.Context, msg *types.MsgClosePosi
 		if err != nil {
 			// Add log about error or not liquidated
 			liqLog = append(liqLog, fmt.Sprintf("Position: Address:%s Id:%d cannot be liquidated due to err: %s", position.Address, position.Id, err.Error()))
+		} else {
+			ctx.EventManager().EmitEvent(sdk.NewEvent(types.EventCloseUnhealthyPosition,
+				sdk.NewAttribute("address", position.Address),
+				sdk.NewAttribute("id", fmt.Sprintf("%d", position.Id)),
+			))
```

Comment:
we don't need this, we already emit this event here `CheckAndLiquidateUnhealthyPosition` and `CheckAndCloseAtStopLoss`

Reply:
we can change the event type in above functions and remove this